homeassistant/components/lutron/binary_sensor.py

```diff
@@ -52,13 +52,11 @@ class LutronOccupancySensor(LutronDevice, BinarySensorEntity):
     _lutron_device: OccupancyGroup
     _attr_device_class = BinarySensorDeviceClass.OCCUPANCY
 
-    @property
-    def is_on(self) -> bool:
-        """Return true if the binary sensor is on."""
-        # Error cases will end up treated as unoccupied.
-        return self._lutron_device.state == OccupancyGroup.State.OCCUPIED
-
     @property
     def extra_state_attributes(self) -> Mapping[str, Any] | None:
         """Return the state attributes."""
         return {"lutron_integration_id": self._lutron_device.id}
+
+    def _update_attrs(self) -> None:
+        """Update the state attributes."""
+        self._attr_is_on = self._lutron_device.state == OccupancyGroup.State.OCCUPIED
```

Comment:
I think this entity should also implement `_request_state` even though it can only be done once (the library doesn't allow force refreshes, just a single first refresh for this device). Otherwise the `update` method won't refresh state during entity addition. Add a comment that it only refreshes state once.

https://github.com/thecynic/pylutron/blob/c75ff6703fe9994bc455faabc8017882f5daa0a6/pylutron/__init__.py#L1206-L1213

Reply:
Nevermind, we already call `_update_attrs` in the `update` method.

Reply:
Uhm, I can add _request_state, but where would this line go then?

Reply:
Yeah, nevermind.